CHANGELOG.md

```diff
@@ -10,6 +10,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add main reconilation loop logic.
```

Comment:
```suggestion
- Add main reconcilation loop logic.
```

Reply:
isn't `reconciliation` the right spelling? :sweat_smile:

Reply:
Haha two typos in the same word!